packages/cozy-clisk/src/launcher/saveFiles.js

```diff
@@ -1,23 +1,89 @@
+// @ts-check
 import Minilog from '@cozy/minilog'
 import get from 'lodash/get'
+import set from 'lodash/set'
```

Comment:
Can't we avoid the use of lodash set? 

Remember: We should go away from lodash. It'll reduce the size of the content script ;)

Reply:
It is allways possible but a lot less elegant and readable I think. But let's do this.

Reply:
Done